integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java

```diff
@@ -64,6 +67,20 @@ public static void generateDbWithTables(Connection connection, int tableCount)
         String.format(
             "Generated %s new constraints:%n%s",
             queryList.size(), Joiner.on(lineSeparator()).join(queryList)));
+    setEnv("HADOOP_SCHEMAS", String.format("%s,%s", DB_SCHEMAS, dbName));
+  }
+
+  private static void setEnv(String key, String value) {
+    try {
+      Map<String, String> env = System.getenv();
+      Class<?> cl = env.getClass();
+      Field field = cl.getDeclaredField("m");
```

Comment:
I don't quite understand why this is necessary. It worries me that this requires knowledge of implementation internals of the used Map class.

Where is the environment variable needed? Can we try to find a solution that does not manipulate the JVMs environment variables?

Reply:
We discussed it a week ago - we need to pass generated schemas to another code execution (dumper), so there were two options via file or env variable. You said, that via env variable will be faster and more convinient, so I implemented it. Should I change it to file or leave it as it is?

Reply:
Changed to file, please check